e2e/app.po.ts

```diff
@@ -32,3 +32,5 @@ export class Ng2JsonEditorPage {
     return element(by.id(id));
   }
 }
+
+export type Webdriver<T> = webdriver.promise.Promise<T>;
```

Comment:
`WDPromise<T>` to be understand that the type is actually some kind of Promise implementation.